client/components/layout/internal/Base/__tests__/Base-spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import expectReactShallow from 'expect-react-shallow'
+
+import Base from '..'
+
+describe('Base', () => {
+  it('renders children within a div by default', () => {
+    expectReactShallow(<Base children={<h1>Test</h1>} type='grid-block' />)
+      .to.have.exactly.rendered(<div className='grid-block'><h1>Test</h1></div>)
+  })
+
+  it('converts the type prop to a foundation class name', () => {
+    expectReactShallow(<Base type='grid-block' />)
+      .to.have.exactly.rendered(<div className='grid-block' />)
+  })
+
+  it('converts the align prop to a foundation class name', () => {
+    expectReactShallow(<Base align='right' type='grid-block' />)
+      .to.have.rendered(<div className='grid-block align-right' />)
+
+    expectReactShallow(<Base align='left' type='grid-block' />)
+      .to.have.rendered(<div className='grid-block align-left' />)
+  })
+
+  it('converts the shrink prop to a foundation class name', () => {
+    expectReactShallow(<Base shrink type='grid-block' />)
+      .to.have.rendered(<div className='grid-block shrink' />)
+  })
+
+  it('converts the vertical prop to a foundation class name', () => {
+    expectReactShallow(<Base vertical type='grid-block' />)
+      .to.have.rendered(<div className='grid-block vertical' />)
+  })
+
+  it('combines multiple props into the correct foundation classes', () => {
+    expectReactShallow(
+      <Base shrink vertical align='right' type='grid-block' />
+    ).to.have.rendered(
+      <div className='grid-block align-right shrink vertical' />
+    )
+  })
+
+  it('merges the provided className with generated foundation classes', () => {
+    expectReactShallow(
+      <Base align='right' className='foo bar' type='grid-block' />
+    ).to.have.rendered(
+      <div className='grid-block foo bar align-right' />
```

Comment:
A lot of these specs have extra classes that we don't care about.  Should we fix that by demonstrating the use of the `shallowRender` utility and matching only parts of the className?